doc/configuration.rst

```diff
@@ -332,6 +332,26 @@ In the example below the special item has ID *DOC-NOTIFICATION*.
         'undefined-reference': 'DOC-NOTIFICATION',
     }
 
+
+.. _traceability_optional_mandatory:
+
+-------------------------------------------------
+Item callbacks, set optional attributes mandatory
+-------------------------------------------------
+
+When using the traceability plugin callback functionality on each item, a combination of 2 attributes
+`non_functional` and `functional` in an item can be made mandatory in the example below.
+
+.. code-block:: python
+
+    item = collection.get_item(name)
```

Comment:
how about you add also function name and that this is to be put into Sphinx `conf.py`

Reply:
and maybe we move this into section on line 285?

Reply:
https://github.com/melexis/sphinx-traceability-extension/pull/229/commits/929ae647677c6630d2e13c02ca7ee426895ecb74

Reply:
This doc file is about the config. No need to repeat.